app/src/main/java/org/fossasia/phimpme/accounts/AccountActivity.java

```diff
@@ -251,11 +251,12 @@ public void onItemClick(final View childView, final int position) {
 
         case DROPBOX:
           if (CLOUDRAIL_APP_KEY == null || CLOUDRAIL_APP_KEY.equals("")) {
-            Snackbar.make(
+            Snackbar snackbar =
+                SnackBarHandler.show(
                     findViewById(android.R.id.content),
-                    R.string.Cloudrail_License_key,
-                    Snackbar.LENGTH_SHORT)
-                .show();
+                    getString(R.string.Cloudrail_License_key),
+                    Snackbar.LENGTH_SHORT);
+            snackbar.show();
```

Comment:
This is ridiculous. Why calling show 2 times. It should only be called once

Reply:
The earlier `.show` is `SnackBarHandler.show()`. This creates a snackbar and returns it. The to show it, we use the second `.show()`.

Reply:
Then it should not be called show. It should be create or make

Reply:
Ok. I'll do that and update the pr.

Reply:
Done.